specs/pitch.md

```diff
@@ -0,0 +1,22 @@
+# MyCanteen
+
+## Problem statement
+
+Cafeterias are popular among students and office workers going for lunch or dinner. Since people tend to eat at the same time, lunchrooms are usually cramped. Customers have to wait in longer queues and look for empty seats afterwards. Also, visitors have to come to cafeteria to find out what is served, how long will they wait etc. Owners have to handle customer spikes during lunchtime while cafes are empty at different hours.
```

Comment:
longer => long || longer and longer
why do you call these places as cafes/cafeterias?

Reply:
Fixed.
These are the same, I think it may be British vs American English thing.